gulpfile.js

```diff
@@ -112,7 +112,9 @@ function createWebpackConfig(defines, output) {
       loaders: [
         {
           loader: 'babel-loader',
+          exclude: /src\/core\/(glyphlist|unicode)/, // babel is too slow
           options: {
+            presets: ['es2015'],
```

Comment:
Agree, is there a preset for somewhat stable stuff implemented by Firefox and Chrome? We can raise this as an issue later.